Source/ASDisplayNode+Layout.mm

```diff
@@ -127,25 +128,20 @@ - (ASTraitCollection *)asyncTraitCollection
 
 ASPrimitiveTraitCollectionDeprecatedImplementation
 
-@end
-
-#pragma mark -
 #pragma mark - ASLayoutElementAsciiArtProtocol
 
-@implementation ASDisplayNode (ASLayoutElementAsciiArtProtocol)
-
 - (NSString *)asciiArtString
 {
   return [ASLayoutSpec asciiArtStringForChildren:@[] parentName:[self asciiArtName]];
 }
 
 - (NSString *)asciiArtName
 {
-  NSString *string = NSStringFromClass([self class]);
+  NSMutableString *result = [NSMutableString stringWithCString:object_getClassName(self) encoding:NSASCIIStringEncoding];
```

Comment:
This is interesting - it seems plausible to be faster, but is that indeed the reason you changed this?